packages/kbn-optimizer/limits.yml

```diff
@@ -124,7 +124,7 @@ pageLoadAssetSize:
   visTypeGauge: 24113
   unifiedSearch: 104869
   data: 454087
-  expressionXY: 26500
+  expressionXY: 44598
```

Comment:
Seems like we can reduce this limit a bit

Reply:
Done. Reduced to `43281`.

Reply:
If you have some other ideas about that, how to reduce it even more, please, suggest it.

Reply:
Page load bundle - 27.6KB. Not so bad 😃

Reply:
Updated expressionXY limit to `29000`.